Packs/CommonScripts/Scripts/Jmespath/jmespath_transformer.py

```diff
@@ -0,0 +1,24 @@
+import demistomock as demisto
+from CommonServerPython import *
+import jmespath
+
+
+def jmespath_search(expression: str, value: dict) -> dict:
+    try:
+        expression_compiled = jmespath.compile(expression)
+    except Exception as err:
+        return_error(f"Invalid expression - {err}")
+    result = expression_compiled.search(value)
+    return result
+
+
+def main():
+    args = demisto.args()
+    value = args.get("value")
```

Comment:
Where is the "value" argument coming from?

Reply:
Value is passed into a transformer script by default. Does it still need to be defined in the YML file? Most other transformer scripts to not require this to be defined.

Reply:
Better to have it set also in the yml file.